src/ec/packages/ec-component-accordion/ecl-accordion.html.twig

```diff
@@ -54,23 +54,35 @@
   {% if _items is not empty %}
     {% for _item in _items %}
       <h{{ _item.level|default(3) }} class="ecl-accordion__title">
-        {% include '@ecl-twig/ec-component-button/ecl-button.html.twig' with {
-          label: _item.toggle.label,
-          variant: 'ghost',
-          type: 'button',
-          icon: _item.toggle.icon,
-          icon_position: 'before',
-          extra_classes: 'ecl-accordion__toggle',
-          extra_attributes: [
-            { name: 'aria-controls', value: _item.id },
-            { name: 'data-ecl-accordion-toggle' },
-          ]
-        } only %}
+        <button
+          class="ecl-button ecl-button--ghost ecl-accordion__toggle"
+          type="button"
+          aria-controls="{{ _item.id }}-content"
+          data-ecl-accordion-toggle
+        >
+          <span class="ecl-button__container">
+            {% include '@ecl-twig/ec-component-icon/ecl-icon.html.twig' with {
+              icon: {
+                path: _item.toggle.icon.path,
+                type: "general",
+                name: _item.toggle.icon.name,
+                size: "s"
+              },
+              extra_classes: 'ecl-icon ecl-icon--s ecl-button__icon ecl-button__icon--before',
+              extra_attributes: [{ name: "data-ecl-icon" }]
```

Comment:
I'm seeing this attribute also in other components including an icon, are we sure this is not a default attribute of this component, now?

Reply:
Maybe, not in the ecl librarie the data attribute isn't defined either on the standard icon. It is just set  on certain icons. I don't know if we should fix this is both libraries or just leave it like this.